pkg/controller/jobs/pod/pod_controller_test.go

```diff
@@ -721,7 +718,7 @@ func TestReconciler(t *testing.T) {
 					Obj(),
 			},
 			wantWorkloads: []kueue.Workload{
-				*utiltesting.MakeWorkload("test-group", "ns").
+				*utiltesting.MakeWorkload("test-group", "ns").Finalizers(kueue.ResourceInUseFinalizerName).
```

Comment:
Note to other reviewers:
The Workload will be finalized in a follow up reconcile, because now it has a Finished condition.